pkg/ui/src/util/api.ts

```diff
@@ -309,3 +312,9 @@ export function getCommandQueue(req: CommandQueueRequestMessage, timeout?: momen
 export function getSettings(_req: SettingsRequestMessage, timeout?: moment.Duration): Promise<SettingsResponseMessage> {
   return timeoutFetch(serverpb.SettingsResponse, `${API_PREFIX}/settings`, null, timeout);
 }
+
+// getReplicaMatrix returns data distribution information
+// TODO(vilterp): rename to DataDistribution?
```

Comment:
do this (here and on backend) before merging. "data distribution" is a more external-friendly name, so we should prob use that.